plugins/by-name/lint/default.nix

```diff
@@ -1,20 +1,18 @@
-{
-  lib,
-  helpers,
-  config,
-  pkgs,
-  ...
-}:
-with lib;
+{ lib, ... }:
 let
-  cfg = config.plugins.lint;
+  inherit (lib)
+    mkOption
+    mkRemovedOptionModule
+    mkRenamedOptionModule
+    types
+    ;
+  inherit (lib.nixvim) defaultNullOpts toLuaObject;
 
   linterOptions = with types; {
```

Comment:
I'd like to avoid having `with` used with too wide a scope.

I'm fine with using it at a "final"/"leaf node" assignment,[^1] but I'm less comfortable having it before a large attrset.

[^1]: e.g. `type = with types; foo bar baz` is ok, but `with types; { foo = "bar"; baz = 123; # ...` is not ideal.